docs/roles/vice-president.rst

```diff
@@ -12,22 +12,27 @@ responsibilities and role of the vice president.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
+The Vice President's primary role is to assist the president where possible. This generally includes helping write and send out emails through the github.io repo as well as helping to manage the presentation schedule. These tasks are generally split evenly between the President and Vice Presient. Making sure these two items continue smoothly is vital and is the thus is the primary responsiblity. 
```

Comment:
Vice Presient should be Vice President. Last sentence should read: "...is vital and thus is the primary responsibility."